src/main/java/edu/usf/cutr/gtfsrtvalidator/api/resource/GtfsFeed.java

```diff
@@ -173,6 +174,12 @@ public Response postGtfsFeed(@FormParam("gtfsurl") String gtfsFeedUrl) {
         if(canReturn)
             return Response.ok(gtfsFeed).build();
         
+        if("checked".equalsIgnoreCase(enableValidation))
```

Comment:
Two code convention issues - can you please add a space here between the `if` and `(`?  Also, please always use `{` for IF statements even if the contents are a single line of code.  Don't worry about changing the other code nearby that doesn't conform, just format the new code you're adding.